ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vxlan-interface.j2

```diff
@@ -1,21 +1,20 @@
 {# eos- VxLAN interface #}
-{% if vxlan_tunnel_interface is defined %}
+{% if vxlan_tunnel_interface is arista.avd.defined %}
 !
 interface Vxlan1
    vxlan source-interface {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
-{%     if vxlan_tunnel_interface.Vxlan1.virtual_router is defined %}
+{%     if vxlan_tunnel_interface.Vxlan1.virtual_router is arista.avd.defined %}
    vxlan virtual-router encapsulation mac-address {{ vxlan_tunnel_interface.Vxlan1.virtual_router.encapsulation_mac_address }}
 {%     endif %}
-   vxlan udp-port {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port }}
+   vxlan udp-port {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port | arista.avd.default('4789') }}
 {%     if vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans is iterable %}
 {%          for vlan in vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans | arista.avd.natural_sort %}
    vxlan vlan {{ vlan }} vni {{ vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans[vlan].vni }}
 {%          endfor %}
 {%     endif %}
-{% if vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs is defined and
-vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs is not none %}
-{%     for vrf in vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs | arista.avd.natural_sort %}
+{%     if vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs is arista.avd.defined %}
+{%         for vrf in vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs | arista.avd.natural_sort %}
    vxlan vrf {{ vrf }} vni {{ vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs[vrf].vni }}
```

Comment:
missing check for `vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vrfs[vrf].vni`